src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star stars__star--rm-margin"></div>
```

Comment:
The 'stars__star--rm-margin' class should be applied to all 'stars__star' elements to remove the default margin.